test/util/mapbox.cpp

```diff
@@ -7,34 +7,65 @@
 
 using namespace mbgl;
 
-// TODO: correct all EXPECT_EQ(actual, expected) to EXPECT_EQ(expected, actual)
 
 TEST(Mapbox, SourceURL) {
-    EXPECT_EQ(mbgl::util::mapbox::normalizeSourceURL("mapbox://user.map", "key"), "https://api.mapbox.com/v4/user.map.json?access_token=key&secure");
-    EXPECT_EQ(mbgl::util::mapbox::normalizeSourceURL("mapbox://user.map", "token"), "https://api.mapbox.com/v4/user.map.json?access_token=token&secure");
-    EXPECT_THROW(mbgl::util::mapbox::normalizeSourceURL("mapbox://user.map", ""), std::runtime_error);
+    EXPECT_EQ(
+        "https://api.mapbox.com/v4/user.map.json?access_token=key&secure",
+        mbgl::util::mapbox::normalizeSourceURL("mapbox://user.map", "key"));
+    EXPECT_EQ(
+        "http://path",
+        mbgl::util::mapbox::normalizeSourceURL("http://path", "key");
+    EXPECT_THROW(
+        std::runtime_error,
+        mbgl::util::mapbox::normalizeSourceURL("mapbox://user.map", ""));
```

Comment:
Unlike `EXPECT_EQ`, the order of the parameters in `EXPECT_THROW` matter: the error type needs to be in the second parameter, not the first.